_posts/en/2016-08-02-copy-to-clipboard.md

```diff
@@ -0,0 +1,25 @@
+---
+layout: post
+
+title: Copy to Clipboard
+tip-number: 55
+tip-username: loverajoel
+tip-username-profile: https://twitter.com/loverajoel
+tip-tldr: This week I had to create a common "Copy to Clipboard" button, I've never created one before and I want to share with I made it.
+
+categories:
+    - en
+---
+
+This is a simple tip, this week I had to create a common "Copy to Clipboard" button, I've never created one before and I want to share with I made it.
+It's easy, the bad thing is that we must have to add an `<input/>` with the text to be copied to the DOM. Then, we selected the content and executes the copy command with [execCommand](https://developer.mozilla.org/en-US/docs/Web/API/Document/execCommand).
```

Comment:
"must have to add" does not work in this context, better would be "we must add" or "we have to add"